test/uk/gov/hmrc/agentsubscriptionfrontend/controllers/FieldMappingsSpec.scala

```diff
@@ -170,35 +173,256 @@ class FieldMappingsSpec extends UnitSpec with EitherValues {
         bind("    ").left.value should contain only FormError("testKey", "error.required")
       }
 
-      "fewer than 10 digits" in {
-        shouldRejectFieldValueAsInvalid("12345         ")
-
-      }
       "more than 24 characters" in {
         shouldRejectFieldValueAsInvalid("999999999999999999999999999999999")
```

Comment:
The error you get when the telephone number is too long isn't very helpful because it doesn't tell you that the reason your input was rejected was that it was too long. Perhaps it should be rejected with an `error.maxLength` in that case, just like agencyName and the address fields?